yarn.lock

```diff
@@ -7,6 +7,17 @@ govuk-frontend@^3.5.0:
   resolved "https://registry.yarnpkg.com/govuk-frontend/-/govuk-frontend-3.5.0.tgz#040e6fd0227f65d3e5820f8a05a9932ffdfb2a9e"
   integrity sha512-4tNKgcChO1bMNsrTz2UsK4fDMmU9R87UDxy/KhKIMbnhsuJLVuArDveYLkZuUsZ6B3eaCbl5gmI8i/Q/Mi680A==
 
+"govwifi-shared-frontend@https://github.com/alphagov/govwifi-shared-frontend/releases/download/0.6.3/govwifi-shared-frontend-0.6.3.tgz":
+  version "0.6.3"
+  resolved "https://github.com/alphagov/govwifi-shared-frontend/releases/download/0.6.3/govwifi-shared-frontend-0.6.3.tgz#6513c83caffbea007c6a7fef8fb89c7e48959166"
+  dependencies:
+    js-cookie "^2.2.1"
+
 html5shiv@^3.7.3:
   version "3.7.3"
   resolved "https://registry.yarnpkg.com/html5shiv/-/html5shiv-3.7.3.tgz#d78a84a367bcb9a710100d57802c387b084631d2"
+
+js-cookie@^2.2.1:
```

Comment:
Are we using this?

Reply:
in the shared-frontend: https://github.com/alphagov/govwifi-shared-frontend/tree/0.6.1

Reply:
sorry wrong link: https://github.com/alphagov/govwifi-shared-frontend/releases/tag/0.6.1

Reply:
So we don't need this again in this repo then?

Reply:
The lock file will include all of them.